drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -186,7 +183,7 @@ def studio_view(self, context):
 
         js_templates = loader.load_unicode('/templates/html/js_templates.html')
         help_texts = {
-            field_name: self._(field.help)
+            field_name: self.ugettext(field.help)
```

Comment:
@asadiqbal08 This might help with the strings in `js_templates.html`: [Customizing the makemessages command](https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#customizing-makemessages).